src/js/component.js

```diff
@@ -98,6 +98,10 @@ class Component {
     this.childIndex_ = {};
     this.childNameIndex_ = {};
 
+    this.setTimeoutIds_ = [];
```

Comment:
Sounds like these would be better served as `Set`s?

Reply:
Yeah that would make sense.